dev/README.md

```diff
@@ -2,7 +2,7 @@ This directory contains resources that the Flutter team uses during
 the development of cocoon.
 
 ## Luci builder file
-`cocoon_try_builders.json` contains the supported luci try builders 
+`try_builders.json` contains the supported luci try builders 
```

Comment:
Is the change on the recipes already running the test?

Reply:
The cocoon recipe is running the json check test now.
With this PR removing the json check codes, the cocoon recipe will stop running the check without any change.
Then I will submit a cl to enable json check from cocoon recipe side by calling the json util.